ngraph/core/include/openvino/core/descriptor/tensor.hpp

```diff
@@ -40,6 +40,7 @@ class OPENVINO_API Tensor {
     void set_name(const std::string& name);
 
     const std::unordered_set<std::string>& get_names() const;
```

Comment:
Can we introduce sth like `constains_name` and reuse it in all palaces where `unordered_set` is iterated to check if the tensor with the given name exists?

Reply:
Sorry, it is my mistake. I removed all redundant cycles and start to use `.count()` method